oathlab/app/src/main/java/com/korbkenny/oathlab/MainActivity.java

```diff
@@ -0,0 +1,160 @@
+package com.korbkenny.oathlab;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.google.gson.Gson;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.FormBody;
+import okhttp3.Headers;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+public class MainActivity extends AppCompatActivity {
+
+    public static final String API_URL = "https://api.twitter.com";
+    public static final String GET_BEARER = "https://api.twitter.com/oauth2/token";
+    public static final String BEARER_TOKEN_CREDENTIALS =
+            "UUo5elJocFZ3REFmZFpkM0hrUTg3YTRTbTowUmZESzQ4bExmMWN0d25RaG55ZEd0UXNzcVhMR2FZT2JSNVJOSmZWY0tSN0FqbXRROQ==";
```

Comment:
You should do your base64 encoding programmatically, rather than hard coding it, so that if you ever need to change your consumer key or consumer secret you can just drop in the new values